pjlib-util/src/pjlib-util/xml.c

```diff
@@ -307,7 +308,12 @@ PJ_DEF(int) pj_xml_print(const pj_xml_node *node, char *buf, pj_size_t len,
 	prolog_len = prolog.slen;
     }
 
-    printed = xml_print_node(node, 0, buf+prolog_len, len-prolog_len) + prolog_len;
+    print_node = xml_print_node(node, 0, buf+prolog_len, len-prolog_len);
```

Comment:
Not introduced by you, but this should really have spaces surrounding the subtraction symbols:

`print_node = xml_print_node(node, 0, buf + prolog_len, len - prolog_len);`

This is because it makes it easier to read as it makes it easier to see visually what variables it is using.